packages/streaming-image-volume-loader/src/StreamingImageVolume.ts

```diff
@@ -415,12 +414,14 @@ export default class StreamingImageVolume extends ImageVolume {
       const options = {
         // WADO Image Loader
         targetBuffer: {
-          arrayBuffer,
+          arrayBuffer:
+            arrayBuffer instanceof ArrayBuffer ? undefined : arrayBuffer,
```

Comment:
Can you please add a comment to explain why we need it for sharedArrayBuffer (since the .set is happening inside the worker in cswil), but not for the arrayBuffer since it happens down below?

Reply:
Sure, I’ll add something like: keeping this in the options means a large empty volume array buffer will be transferred to the worker. 

This is undesirable for streaming volume without shared array buffer because the target is now an empty 300-500MB volume array buffer. Instead the volume should be progressively set in the main thread.